configurations/admin.template.ini

```diff
@@ -1220,6 +1220,18 @@ moduls.teamsRecordingUploadIntegration.label = "Enable Teams recordings upload i
 moduls.teamsRecordingUploadIntegration.permissionName = FEATURE_TEAMS_RECORDING_UPLOAD_PERMISSION
 moduls.teamsRecordingUploadIntegration.group = GROUP_ENABLE_DISABLE_FEATURES
 
+moduls.cncAI.enabled = true
+moduls.cncAI.permissionType = 2
+moduls.cncAI.label = "Enable CNC Assistant"
+moduls.cncAI.permissionName = FEATURE_CNC_ASSISTANT_PERMISSION
+moduls.cncAI.group = GROUP_ENABLE_DISABLE_FEATURES
+
+moduls.cncAssistant.enabled = true
+moduls.cncAssistant.permissionType = 2
+moduls.cncAssistant.label = "Enable AI in CNC - Sentiment Analysis"
```

Comment:
you need to flip your permissions. you used cncAssistant for AI and AI for assitent.
also dont call the permissions cncAI as we might have more in the future, call it cncAIAssistent